rules/android_local_test/attrs.bzl

```diff
@@ -175,9 +175,10 @@ def make_attrs(additional_aspects = [], native_libs_transition = None):
             env = attr.string_dict(
                 doc = "A dictionary of environment variables set for the execution of the test. Will be subject to make variable and $(location) expansion.",
             ),
-            robolectric_properties_file = attr.string(
+            robolectric_properties_file = attr.label(
```

Comment:
You can actually just look up the properties file in the runfiles and error out if they aren't found. I believe this is how the native Android rules handle this use case already.

https://github.com/Bencodes/rules_kotlin/blob/blee/fork/master-napt-1.8-rules-android-pre-alpha/kotlin/internal/jvm/kt_android_local_test_impl.bzl#L178-L183

Reply:
Thanks for the example. But if user have the option to use their own properties file, would this look-up approach work? On the other hand, I'm not sure if users ever want to use their own properties files.

Reply:
It should as long as you are writing the file to the expected location. Probably isn't really a major use case that people will have though.

One use case that comes to mind would be stripping out old Android versions that you no longer want to support from Robolectric. If there is a need to customize the properties file we can always build out support for that in `rules_robolectric`.